BUILD.bazel

```diff
@@ -0,0 +1,78 @@
+package(default_visibility = ["//visibility:public"])
+
+# TODO(jelbourn): figure out if these workarounds are still needed
```

Comment:
don't add TODOs for Jeremy in this repo :)

Reply:
Since this falls under Material, I'll keep it in, though it's not likely he'll ever see it 😉